free-3.md

```diff
@@ -57,12 +74,15 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+This seems like a project that could actually have a lot of cool language design. That said, a lot of initial work would have to be done to teach or input rules of various poems to the system, and then work would have to be done on the checker, both of which seem to be substantial tasks. After that, though, all sorts of additional functionality and design options open up; would you want to maybe have the language be able to suggest rhymes, alliterations, and other thing? Maybe we would want to allow users to define a new poem type (rhyme scheme, meter, length etc) that could then be checked by the checker? It seems like possibilities abound here.
```

Comment:
Considering how a poem is defined is important. If you choose how users will define poems carefully, you, as the designer, can use this interface for defining the predefined structures as well.